pkgs/data/icons/nordzy-icon-theme/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, bash
+, gtk3
+, nordzy-themes ? [ "all" ] # Override this to only install selected themes
+}:
+
+let
+  themes-arg-string = lib.strings.concatMapStrings (theme: "-t ${theme} ") nordzy-themes;
+in
+stdenv.mkDerivation rec {
+  pname = "nordzy-icon-theme";
+  version = "unstable-2021-12-14";
+
+  src = fetchFromGitHub {
+    owner = "alvatip";
+    repo = "Nordzy-icon";
+    rev = "5c247a4f19cf9849615631d1bf77727b945b634e";
+    sha256 = "Jqn5CF80xlYJ7H4qI1VEj91vcKPPoMXP5+sPs0ksiC4=";
+  };
+
+  nativeBuildInputs = [ gtk3 ];
```

Comment:
Is this really needed? There's no real program code here that has to be linked against gtk3

Reply:
Well you need `gtk3` to run `gtk-update-icon-cache` which is being called in `install.sh`. Is there another way you would do this?

Reply:
Is that gtk cache even used? I thought another one was built when the system path is assembled. Maybe you can just not generate it here

Reply:
Maybe we want to set `dontDropIconThemeCache = true;`?